src/Common/src/CoreLib/System/String.Searching.cs

```diff
@@ -473,4 +473,4 @@ public int LastIndexOf(string value, int startIndex, int count, StringComparison
         [StructLayout(LayoutKind.Explicit, Size = PROBABILISTICMAP_SIZE * sizeof(uint))]
         private struct ProbabilisticMap { }
     }
-}
+}
```

Comment:
The CoreCLR copy of the file does not seem to have missing end of line. Is this change correct?